scripts/tests/test_validate_docstrings.py

```diff
@@ -664,6 +715,19 @@ def list_incorrect_parameter_type(self, kind):
         """
         pass
 
+    @Decorators.missing_wraps
+    def bad_decorator(self, kind: bool):
```

Comment:
Sorry I didn't think about this in the previous reviews, but this feels a bit wrong. We're testing that we're unable to get the correct signature for this.

Since I guess we can't know whether the documented parameters are wrong, or the function is incorrectly wrapped, I think probably the best is simply to remove this test. If this test fails would be great news, so I don't think we want to have it. Or we can have it as an `xfail`.

@WillAyd what do you think?

Reply:
Imo xfail would probably be best. This test demonstrates the problem that was solved, and it passing means that the script should be worked on again to remove extraneous code.

Reply:
I may be missing something, but if we had the function `bad_decorator` in pandas, what we should document in Parameters is `kind`, right? Since our validation will fail if `kind` is documented, I'd say that we solved other problems, but this test is something that ideally we'd like to fix.

Am I wrong?

Reply:
I think you're right, I'm just not sure how I'd go about fixing it. Which error would we like to produce for the test?

Reply:
I'd probably just delete the test. Or as I said, I'd move it to the passing tests, and then xfail it. But probably not worth.

Reply:
Alright, I'll remove it and push another commit.